### ISSUE TITLE
printf format mismatch in "xcf: use SDL_PRIx macros in debug messages"?

### ISSUE THREAD

**Issue by smcv** [libsdl-org/SDL_image#411]
6d91bf78 has this:

```diff
     }
 #if DEBUG
-    SDL_Log("%.8" SDL_PRIs64 ": %s(%u): %u\n", SDL_RWtell (src), prop->id < 25 ? prop_names [prop->id] : "unknown", prop->id, prop->length);
+    SDL_Log("%.8" SDL_PRIu32 ": %s(%u): %u\n", SDL_RWtell (src), prop->id < 25 ? prop_names [prop->id] : "unknown", prop->id, prop->length);
 #endif
 
     switch (prop->id) {
```

which seems wrong? `SDL_RWtell()` returns `Sint64`, so I would have expected `SDL_PRIs64` to be correct.

(This is a drive-by comment while updating the snapshot of SDL3 in Debian experimental, I haven't actually tried compiling with debug enabled.)

cc @madebr

**Comment by madebr** [comment]
Oops. Thanks for the drive-by comment :)

**Comment by madebr** [comment]
I fixed this in 2487900
